tests/unit/song_tests.js

```diff
@@ -0,0 +1,42 @@
+describe('song tests', function() {
+    var songObj = {"image":"../core/assets/2015-best-music.png","songTitle":"love love enemies love","songAuthor":"Smiley","songLength":74,"songListened":82};
+    var song;
+    beforeEach(function() {
+      song = new Song(songObj);
```

Comment:
You can do here(to avoid the risk of sharing "state" between tests) this.song = new Song() and in every test you can access it as this.song